Library/Edge.cs

```diff
@@ -73,9 +83,12 @@ public void SetNext(Vertex v, Edge other)
             else next2 = other;
         }
 
-        /**
-         * Similar to Next() but to go backward in the double-linked list
-         */
+        /// <summary>
+        /// Returns the previous <see cref="Edge"/> in the linked list of edges that use the specified <see cref="Vertex"/>.
```

Comment:
It's important imho to mention that it is the opposite of `Next` so that the reader can think "oh ok I understood Next() so I don't even have to read all this, I get it".